contracts/finality/src/exec/finality.rs

```diff
@@ -331,12 +325,11 @@ fn ensure_fp_exists_and_not_slashed(deps: Deps, fp_pubkey_hex: &str) -> Result<(
 /// `slash_finality_provider` slashes a finality provider with the given evidence including setting
 /// its voting power to zero, extracting its BTC SK, and emitting an event
 fn slash_finality_provider(
-    env: &Env,
     info: &MessageInfo,
-    fp_btc_pk_hex: &str,
     evidence: &Evidence,
-) -> Result<(WasmMsg, Event), ContractError> {
-    let pk = eots::PublicKey::from_hex(fp_btc_pk_hex)?;
+) -> Result<(BabylonMsg, Event), ContractError> {
```

Comment:
If we rebase this PR, the `BabylonMsg` is no longer imported, due to https://github.com/babylonlabs-io/rollup-bsn-contracts/pull/23

Reply:
resolved